plugins/node/instrumentation-runtime-node/src/metrics/eventLoopUtilizationCollector.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { EventLoopUtilization, performance } from 'node:perf_hooks';
+import { RuntimeNodeInstrumentationConfig } from '../types';
+import { Meter } from '@opentelemetry/api';
+import { BaseCollector } from './baseCollector';
+
+const { eventLoopUtilization: eventLoopUtilizationCollector } = performance;
+
+export const NODEJS_EVENT_LOOP_UTILIZATION = 'eventloop.utilization';
+
+export class EventLoopUtilizationCollector extends BaseCollector<EventLoopUtilization> {
+  constructor(
+    config: RuntimeNodeInstrumentationConfig = {},
+    namePrefix: string
+  ) {
+    super(config, namePrefix);
+  }
+
+  public updateMetricInstruments(meter: Meter): void {
+    meter
+      .createObservableGauge(
+        `${this.namePrefix}.${NODEJS_EVENT_LOOP_UTILIZATION}`,
+        {
+          description: 'Event loop utilization',
+          unit: 's',
+        }
+      )
+      .addCallback(async observableResult => {
+        if (this._scrapeQueue.length === 0) {
+          return;
+        }
+        const elu = eventLoopUtilizationCollector(this._scrapeQueue.shift());
+        observableResult.observe(elu.utilization);
```

Comment:
Rather than reporting a gauge we found it useful to change this to a cumulative counter and just report the Active/Idle seconds. Once again we eliminated the scrape queue from this code path.

With the cumulative utilization in seconds we can use the rate function in prometheus to see the % utilization easily. Even if we retain the utilization as a gauge here we should remove the scrape queue as it suffers from the same problem as event loop delay where if the measurement precision is not aligned with your metric export interval the scrape queue will just leak entries and your exported metrics will not be aligned with reality.

Reply:
As i see in convention it's still Gauge

Reply:
@maryliag what do you think ?

Reply:
Yeah using the cumulative counter would be more aligned with the cpu time metrics on process:
https://opentelemetry.io/docs/specs/semconv/system/process-metrics/#metric-processcputime

We almost exclusively use cpu time metrics rather than cpu utilization metrics, but we could definitely report both active time and idle time for event loop + utilization for people with different preferences.

Reply:
In case it helps here was the implementation we did locally, perhaps naming would need to be updated to reflect the spec:
```ts
 public updateMetricInstruments(meter: Meter): void {
    const active = meter.createObservableCounter(`${this._namePrefix}.${NODEJS_EVENT_LOOP_UTILIZATION_ACTIVE}`, {
      description: 'Event loop utilization active total seconds',
      unit: 's',
    });
    const idle = meter.createObservableCounter(`${this._namePrefix}.${NODEJS_EVENT_LOOP_UTILIZATION_IDLE}`, {
      description: 'Event loop utilization idle total seconds',
      unit: 's',
    });

    meter.addBatchObservableCallback(
      async observableResult => {
        if (this._enabled === false) {
          return;
        }

        const elu = eventLoopUtilizationCollector();
        observableResult.observe(active, elu.active / 1000, versionAttribute);
        observableResult.observe(idle, elu.idle / 1000, versionAttribute);
      },
      [active, idle]
    );
  }
```

Reply:
I think it's good but we need to synchronize this behavior with convention

Reply:
the change sounds good to me, make sense to sync with the `process.cpu.time`. I can make the update on the convention itself

Reply:
Created the PR with the update, waiting for approval: https://github.com/open-telemetry/semantic-conventions/pull/1259

Reply:
@maryliag left a comment on your PR I don't think utilization can be directly represented as a counter we would need to report active / idle seconds.

Reply:
add new counters

Reply:
this seems resolved as well, please confirm